aspnet/web-pages/overview/security/using-a-catpcha-to-prevent-automated-programs-bots-from-using-your-aspnet-web-site.md

```diff
@@ -34,11 +34,11 @@ You can help make sure that a user is real person and not a computer program by
 
 ## Adding a ReCaptcha Test
 
-In ASP.NET pages, you can use the `ReCaptcha` helper to render a CAPTCHA test that is based on the ReCaptcha service ([http://recaptcha.net](http://recaptcha.net)). The `ReCaptcha` helper displays an image of two distorted words that users have to enter correctly before the page is validated. The user response is validated by the ReCaptcha.Net service.
+In ASP.NET pages, you can use the `ReCaptcha` helper to render a CAPTCHA test that is based on the [ReCaptcha service](https://www.google.com/recaptcha/about/). The ReCaptcha helper displays an image of two distorted words that users have to enter correctly before the page is validated. The user response is validated by the ReCaptcha.Net service.
```

Comment:
Your link is a decent suggestion, but given the audience, this one might be preferable: https://developers.google.com/recaptcha/